src/devices/video/ef9365.cpp

```diff
@@ -745,9 +753,12 @@ int ef9365_device::draw_vector(int x1,int y1,int x2,int y2)
 
 			x++;
 
-		}while( x <= x2 );
+		}while( x <= dest_x );
```

Comment:
Spacing is weird here too. Our usual syntax would be:

`} while (x <= dest_x);`